pkg/dfc/dfc.go

```diff
@@ -958,15 +982,29 @@ func processRunLine(newLine *DockerfileLine, line *DockerfileLine, stagePackages
 		runPrefix := DirectiveRun + " "
 		runIndex := strings.Index(upperRawLine, runPrefix)
 
+		var defaultConverted string
 		if runIndex != -1 {
 			// Get the original case of the RUN directive
 			originalRunDirective := rawLine[runIndex : runIndex+len(runPrefix)]
-			newLine.Converted = originalRunDirective + afterShell.String()
+			defaultConverted = originalRunDirective + afterShell.String()
 		} else {
 			// Fallback if we can't find the directive (shouldn't happen)
-			newLine.Converted = DirectiveRun + " " + afterShell.String()
+			defaultConverted = DirectiveRun + " " + afterShell.String()
+		}
+
+		if runLineConverter != nil {
+			custom, err := runLineConverter(newLine.Run, defaultConverted, line.Stage)
+			if err != nil {
+				return err
+			}
+			if custom != "" {
+				newLine.Converted = custom
+				return nil
+			}
```

Comment:
Lets let the end user assign empty string conversion if desired, which would (should) result in skipping conversion
```suggestion
			newLine.Converted = custom
```